frontend/src/App.jsx

```diff
@@ -13,7 +14,29 @@ import SignupSuccess from './components/signupSuccess';
 import Profile from './components/profile';
 import Request from './components/requestOff';
 import Availability from './components/availability/availability';
+import Contacts from './components/contacts';
+import { Fragment } from 'react';
 
+/* only allow admins and employees to see certain pages */
+function determineRoutes(props) {
```

Comment:
Instead of having this function to determine routes, let's change it to Required auth checking user roles to determine whether or not we can send the person to the page. You will have to change requireAuth.js and some other stuff. (I made a couple changes make sure to pull first) You can reference this [video](https://youtu.be/oUZjO00NkhY)